python/cudf/cudf/api/types/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+from .categoricals import _union_categoricals
```

Comment:
We haven't used relative imports anywhere else in the codebase. Could we keep consistent with absolute imports until we decide to do otherwise?

Reply:
Sure